blocks/layout-grid/src/grid-resize.scss

```diff
@@ -31,7 +31,7 @@
 
 	// Every block, including nested blocks, are born with an intrinsic minimum margin.
 	// This resets that for the containers, which aren't meant to have that intrinsically.
-	> .wp-block > .editor-block-list__block-edit > [data-block] {
+	> .wp-block > .block-editor-block-list__block-edit > [data-block] {
```

Comment:
`.editor` as a prefix is deprecated in favor of `.block-editor`, which should have been around for a while.